Weather app/ViewModels/ViewModel.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  ViewModel.swift
+//  Weather app
+//
+//  Created by Ernest Mihasenko on 27.07.22.
+//
+
+import Foundation
+import CoreLocation
+import Combine
+
+class CurrentWeatherViewModel: NSObject, ObservableObject, CLLocationManagerDelegate {
+    
+    @Published var weatherData: NetworkWeatherData?
+    @Published var isLoading = false
+    @Published var presentsSearch = false
+    @Published var city: String = "City"
+    @Published var searchCity: String = ""
+    @Published var temperature: String = "0"
+    @Published var status: String = "Status"
+    @Published var daily = [Daily]()
+    @Published var humidity: String = "Humidity"
+    @Published var windSpeed: String = "Wind speed:"
```

Comment:
Maybe you can create one model that combines all weather data and use it here instead of these many properties? You can pass `NetworkWeatherData` as parameter in init of this model